### ISSUE TITLE
blame view keybindings are hard to discover

### ISSUE THREAD

**Issue by ilyagr** [jonas/tig#1315]
I wish they were documented in the in-program help better. There could be a "blame bindings" section. I realize that all the bindings are mentioned in the help already, but their descriptions make it very difficult to understand how they actually work in blame view. Blame is very useful occasionally, but I find it difficult to remember the bindings whenever I use it.

My understanding is that, in blame view:

- `Enter` opens the blamed commit for the current line in a separate pane
- `b` opens the current file at the blamed commit when already in blame view
- `<` goes back to the previous view, e.g. undoes `b` (as elsewhere in tig, but I only use it when blaming)
- `,` seems like it could be useful, but isn't in blame view

Thanks for making `tig`, it's very useful! I was thinking of making a PR for this, but I couldn't immediately figure out how the help is generated.

**Comment by ilyagr** [comment]
I realized that even here, I misunderstood something. It seems that the `,` command is important in blame. I'm guessing it's meant to go to either (the last commit before the current one that modified the current file) or (the last commit before the blamed commit for the current line that modified the selected line).

One confusing thing about it is that it sometimes shows the message "The selected commit has not parents". I wish this message was clearer: is the selected commit the one on the current line or the one currently being viewed? It should also mention say something like "...has no ancestors that modify this (file/line)".

**Comment by ilyagr** [comment]
BTW, I'd be happy to try implementing this myself, but it'd help if you gave me some pointers.

**Comment by krobelus** [comment]
> , shows the last commit before the blamed commit for the current line that modified the selected line.

Right. I never really used, it seems nice. Sadly it can't know where a line moved to, so the initial position can be confusing.
I usually alternate between blame and diff view for recursive blaming.

**Comment by ilyagr** [comment]
https://github.com/jonas/tig/issues/1315

> Sadly it can't know where a line moved to, so the initial position can be confusing.

That's a good point. If the "going to an ancestor of the currently displayed commit" operation existed, it'd be equivalent to `b`  followed by that operation, so it overall seems less important.

I think having `b`, `B` (optional, would do what `,` currently does), `,` (for going to a parent of the currently displayed commit, as seems to match its description in the help, and is nice to track the evolution of a file), and `<` would be a good interface, especially it's documented. The (new) blame section of the help could document `b` and `B`, and either also document `,` and `<` (repeating an earlier section, possibly with variations), or just mention "See also `,` and `<` commands which are particularly useful in blame view".

We could also keep `,` as is, and add `B` that does the other thing, but I think that's be more confusing for all people except those who are very used to the current behavior of `,`.

For diff view, I get a bit confused by the multi-view controls, so I seem to tend to avoid it. I just realized that `<Tab>` seems to switch the meaning of `j`/`k` and `J`/`K`, but *not* the meaning of Up/Down. But that's a separate issue that might be harder to address.

**Comment by ilyagr** [comment]
I made #1373 to partially address this. I think I'm finally comfortable enough with what these commands do to write it down :)

As an aside or motivation, here's another reference for this being hard to discover: 

https://github.com/gitext-rs/git-dive/blob/main/docs/comparison.md#tig
